### DISCUSSION TITLE
Redis persistence

### DISCUSSION THREAD

**Discussion by defulmere** [mastodon/mastodon#22976] [Q&A] [answered]
Are there any recommended Redis persistence settings for redis with mastodon? Specifically looking for recommendations for `save` intervals and whether or not I should use `appendonly`.

**Comment by themullinator** [comment, marked as answer]
Based on https://docs.joinmastodon.org/admin/scaling/#redis, I'd say the recommendation is no persistence. Loss of the Redis cache is fully survivable, and at least on smaller instances like mine, there's no noticeable performance penalty rebuilding the cache after a host reboot.
